packages/deployment/app/cloudformation.template.yml

```diff
@@ -1005,12 +1005,12 @@ Resources:
     Properties:
       AdjustmentType: PercentChangeInCapacity
       AutoScalingGroupName: !Ref CumulusECSAutoScalingGroup
-      EstimatedInstanceWarmup: 300
+      EstimatedInstanceWarmup: 180
```

Comment:
This is the amount of time that it takes for a new ECS instance to become available in the cluster, so that will be constant no matter what services are running on top of the cluster.